kodilla-collections-advanced/src/main/java/com/kodilla/collections/adv/immutableSpecial/homework/BookApplication.java

```diff
@@ -14,16 +14,8 @@ public static void main(String[] args) {
         System.out.println(bookManager.getBooks().size());
 
 
-        Book b1 = new Book("Kompendium programisty.", "Jurek Jurek");
-        Book b2 = new Book("Kompendium programisty.", "Jurek Jurek");
-        System.out.println("Porównanie adresów: "+(b1 == b2));
-
-//        System.out.println("Porównanie adresów: "+(b1 == b2));
-//        System.out.println("Porównanie adresów: "+(b1.equals(b2)));
-//        System.out.println("Porównanie adresów: "+(b2.equals(b3)));
-//        System.out.println("Porównanie adresów: "+(b1.equals(b4)));
-//        System.out.println("Porównanie adresów: "+(b1.hashCode() == b4.hashCode()));
-//        System.out.println("Porównanie adresów: "+(b1.equals(new ImmutableSpecialCases())));
-
+        Book b1 = new Book ("Kompendium programisty.", "Jurek Jurek");
```

Comment:
a co to daje? Trzeba tworzyc przez BookManagera.

Reply:
Book book1 = bookManager.createBook("Kompendium ...", "Jurek ...");
Book book2 = bookManager.createBook("Kompendium ...", "Jurek ...");
if (book1 == book2 ) { System.out.println("OK"); } else { System.out.println("Something went wrong";}